### PR TITLE
k8s beta signup form

### PR DESCRIPTION
**Pull request by camerondgray**

### CODE REVIEW

**Comment by ddollar** [comment]
Easier to just give out the Google Form links than have to maintain these redirects

**Comment by camerondgray** [comment]
@ddollar I think the convox URL looks way better than the google docs URL (even if it re-directs). It's also possible that people at events will see/remember it and signup. Also, once we are actually live with k8s the URLs can point to landing pages instead of re-directing so they won't need to be maintained.

**Comment by ddollar** [comment]
OK, then make them redirects so we don't leak cookies, and put them in `_config/convox.conf`

File: convox.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
     command: _bin/web
     domain: ${HOST}
     environment:
-      - CONSOLE_HOST
+      - CONSOLE_HOST=
```
Comment (ddollar) [inline convox.yml]: These should not be defaulted to blank. They are there to prevent deploying a broken site.

**Pull request merged**